docs/04-Services/04-Navigation-Manager.md

```diff
@@ -0,0 +1,47 @@
+---
+since: 0.1.1
+---
+
+# Navigation Manager
+
+## Service Overview
+
+This service simplifies keyboard navigation for a set of DOM elements. It exposes a factory function `createNavManager()` that creates an object containing:
+
+- `elementsInDomOrder$` - a store containing the navigable elements in DOM order
+- `directive` - a directive to use on some DOM elements, both to add the keydown event handler and to specify which elements should be managed (either by directly putting the directive on those elements, or by putting the directive on a parent element and specifying which child elements should be included through a `selector` function via `NavManagerItemConfig`)
+- Utilities to navigate the items:
+  - `refreshElements` - method to manually refresh the `elementsInDomOrder$` to capture the latest state of DOM
+  - `focusIndex` - method to focus the element at the given index within `elementsInDomOrder$`
+  - `focusPrevious` - method to focus the previous element, respecting the anscestor direction
```

Comment:
There is a typo with the `ancestor` word (also in the following lines).
I think it is not clear which ancestor we are talking about. I know from the code, but it is not explained in the doc that it is the closest common ancestor of all elements which is considered.

Reply:
Maybe it should be explained that `focusPrevious`, `focusNext`, `focusFirst` and `focusLast` refer to the DOM order, and that `focusLeft`, `focusRight`, `focusFirstLeft` and `focusFirstRight` will be do exactly the same in a LTR configuration and the opposite in a RTL configuration.